Restructuring proposal with some overall changes to the doc (#9513)
mendix/docs: content/en/docs/control-center/apps/apps-overview.md

## Introduction

The **Apps** page in Control Center provides a central place to monitor, manage, and interact with all the apps belonging to your company. 

> Using the search bar at the top of the page, you can find an app using either its name or app ID (which you can find under **Project ID** in your app’s [Settings](/developerportal/collaborate/general-settings/) in [Apps](https://sprintr.home.mendix.com/)).

Reviewer: Perhaps an imperative?
```suggestion
Use the search bar at the top of the page to find an app using either its name or app ID (which you can find under **Project ID** in your app’s [Settings](/developerportal/collaborate/general-settings/) in [Apps](https://sprintr.home.mendix.com/)).
```

{{< figure src="/attachments/control-center/apps/apps-overview/top-search-bar.png" >}}

Search results appear in a pop-up window where you can apply filters for **AppName**, **AppID**, **Technical Contact**, **Last Commit**, **Created Date**, and **Deployment**. Click the app name to open the [app details](#app-details).

> The **Apps** page includes the following tabs: **Mendix Cloud**, **Private Cloud**, **Mendix Free Cloud**, **Other Apps**, and **Deactivated Apps**.

Reviewer: I think "Private Cloud" might have changed to "Mendix for Kubernetes" but perhaps not yet?

Author: It's now referred to as Mendix on K8s, but the UI hasn't been changed yet. Will reach out to the team

Author: Update, UI isn't to be changed for now. So, usage continues as it is.

Each tab contains a table. By clicking the column selector ({{% icon name="view" %}}) in the upper-right corner of the table, you can choose which fields to display. You can also:

* Filter the apps in the table. For details, see the [Filtering Apps in the Table](#filter-apps) section

> * Export app information to an Excel file. For details, see the [Exporting Apps to an Excel File](#export-to-excel) section

Reviewer: Could we put the link to **Fields Glossary** here as well? Then we remove it for each tab?

(It's already great that the information is summarized in one place!)

## Mendix Cloud

The **Mendix Cloud** tab lists all licensed apps that can be deployed on Mendix Cloud.

> By default, the table shows only apps published in the production environments. To view all environments, toggle off the **Production** switch in the upper-right corner of the table.

Reviewer: We tend to use "deploy to" (not sure if that is in the style guide?) But in any case I think we should be consistent with prepositions in "deployed …" and "published …" in these two lines. And perhaps just stick to one verb (boring, but this is technical writing, not a novel 😂

When you select apps using the checkboxes in the table, a context menu appears at the bottom of the screen. Use it to export the selected apps.

See the [Fields Glossary](#fields-glossary) section for descriptions of the table columns.

> ## Mendix on Kubernetes (Connected) {#pc-connected}

Reviewer: Also see naming note below. Need to check the current UX and also discuss with Kasia.

Author: Discussed with Kasia and for CC, Private Cloud naming to remain as it is. UI not to be changed yet

The **Mendix on Kubernetes** tab shows all the apps that are deployed on the Mendix on Kubernetes Connected.

See the [Fields Glossary](#fields-glossary) section for descriptions of the table columns.

## Mendix Free Cloud

The **Mendix Free Cloud** tab shows all the free apps deployed on Mendix Cloud.

See the [Fields Glossary](#fields-glossary) section for descriptions of the table columns.

{{% alert color="info" %}}
When you select apps in the Mendix Free Cloud or the Other Apps tabs using the checkboxes in the table, a context menu appears at the bottom of the screen. Use it to deactivate, delete, or export selected apps. See the [Deactivating and Deleting Apps](#deactivate-delete-apps) section for more details.
{{% /alert %}}

## Other Apps

> The **Other Apps** tab shows unlicensed apps that are not deployed and apps deployed on a Private Cloud Connected .

Reviewer: ```suggestion
The **Other Apps** tab shows unlicensed apps that are not deployed and apps deployed on a Mendix on Kubernetes Connected .
```

Check product name with Kasia.

Author: Product name remains as it is to match with UI.

See the [Fields Glossary](#fields-glossary) section for descriptions of the table columns.

## Deactivated Apps

The **Deactivated Apps** tab shows all your company's deactivated apps.

See the [Fields Glossary](#fields-glossary) section for descriptions of the table columns.

When you select apps using the checkboxes in the table, a context menu appears at the bottom of the screen. Use it to activate, delete, or export selected apps. After activation, the app appears on the corresponding active tab. See the [Deactivating and Deleting Apps](#deactivate-delete-apps) section for more information.

> ## Fields Glossary {#fields-glossary}

Reviewer: Great to pull this out and have it in one place. And very comprehensive, stating which tabs they appear in.

I wonder whether this section, Filtering, Deactivating, and Exporting should perhaps all come in one section (Perhaps "Interacting with App Lists", "Interacting with Tables of App",  or something like that) to make a distinction between the descriptions of the tabs and the descriptions which apply to multiple tabs.

But having them at the top level is OK too.

Author: This is a great suggestion. Makes sense to have a common theme grouping. However, I will be using "App Management" for the heading grouping as I think that fully encompasses the actions in the section.

> This glossary describes all the fields that appear across all the tables in the **Mendix Cloud**, **Private Cloud**, **Mendix Free Cloud**, **Other Apps**, and **Deactivated Apps** tabs.

Reviewer: Private Cloud/Mendix on Kubernetes?

Author: Private Cloud for now due to current UI

| **Field**               | **Description**                                                                                                                                                                      | **Appears In**                                  |
| ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ----------------------------------------------- |

> | **Action**              | The action menu ({{<icon name="three-dots-menu-horizontal">}}) which provide options to **Activate App** and **Delete App**. See the [Deactivating and Deleting Apps](#deactivate-delete-apps) section | Mendix Free Cloud, Other Apps, Deactivated Apps |

Reviewer: ```suggestion
| **Action**              | The action menu ({{<icon name="three-dots-menu-horizontal">}}) which provides options to **Activate App** and **Delete App**. See the [Deactivating and Deleting Apps](#deactivate-delete-apps) section | Mendix Free Cloud, Other Apps, Deactivated Apps |
```

| **Namespace**           | Kubernetes namespace. See [Creating a Private Cloud Cluster](/developerportal/deploy/private-cloud-cluster/#create-cluster-namespace)                                                      |  Private Cloud  (Connected)                         |
| **NamespaceID**         | Namespace ID                                                       |  Private Cloud (Connected)                         |
| **Storage Name**        | Name of the storage                                                                                                                                                         |  Private Cloud                          |
| **Technical Contact**   | Technical contact of the app. To edit, click **Edit** ({{<icon name="pencil">}}) next to the current Technical Contact’s name                                                    |  Mendix Cloud, Mendix Free Cloud, Private Cloud (Connected) |

> ## Filtering Apps in the Table {#filter-apps}

Reviewer: Since there are multiple tables in different tabs, perhaps
```suggestion
## Filtering Apps in a Table {#filter-apps}
```

On each tab with a table of apps, you can find filters on the top, which allows you to view apps based on different criteria.

With the filter as shown below, you can search for string matches on any of the string fields in the table, for example, the app name or environment name.

Clicking the filter button allows you to use comparisons and string operations, such as **Contains**, **Starts with**, and **Greater than**.

{{< figure src="/attachments/control-center/apps/apps-overview/filter-apps.png" >}}

> With the filter as shown below, you can filter apps by the date of the last commit. Clicking the button on the left allows you to use comparisons and string operations, such as **Between** and **Empty**. 

Reviewer: Not string operations on dates?

Author: No. Fixed.

* **Environments** – Lists all Mendix Cloud environments. For the free Mendix Cloud environment, you can delete the environment on this tab. Once the free environment is deleted, all data will be removed. However, the app team members can still access the associated app repository in the Mendix Portal.

If the app is not a licensed app, you can see the action menu ({{% icon name="three-dots-menu-horizontal" %}}) on the upper-right corner of the page. If the app is active, then you have the options to deactivate the app and delete the app; and if the app is deactivated, then you have the options to activate the app and delete the app.

{{% alert color="info" %}}
You cannot deactivate or delete licensed apps or apps not hosted on Mendix Cloud. For the consequences of these operations, see the table in the [Deactivating and Deleting Apps](#deactivate-delete-apps) section for more information.
{{% /alert %}}

## Deactivating and Deleting Apps {#deactivate-delete-apps}

> You can delete or deactivate a single app, or deactivate (or activate) multiple apps in one go.

Reviewer: Perhaps mention "Free Apps" here as well as the info box later, then readers will know immediately if it is relevant to them and their app?
```suggestion
You can delete or deactivate a single free app, or deactivate (or activate) multiple free apps in one go.
```

| Cloud environment stays persistent                          | {{< icon name="remove-circle-filled" color="red" >}} | {{< icon name="remove-circle-filled" color="red" >}} |

{{% alert color="info" %}}
Deleting and deactivating apps is currently not supported for licensed apps or apps that are not running on Mendix Cloud.
{{% /alert %}}

## Exporting the Information About Apps to an Excel File {#export-to-excel}

You can export app data to Excel file from the table in any of the available tabs.

* To export selected apps:

  1. Select the apps using the checkboxes in the table.
  2. In the context menu that appears at the bottom of the screen, click **Selection Export**.

* To export all apps: Click **Export All** in the upper-right corner of the table.

> The Excel file covers the following columns: **AppId**, **AppName**, **TechnicalContact**, **Environment**, **DeployedStatus**, **MendixVersionOnMainline**, **DeployedMendixVersion**, **LastCommit**, **Members**, **Production**, **AppCreatedDate**, **CloudLocation**, **Fallback**, **CloudResourcePack**, **AppRAM**, **DbRAM**, **DbStorage**, **DbStorageExtra**, and **FileStorage**.

Reviewer: Perhaps "includes" rather than "covers"?
```suggestion
The Excel file includes the following columns: **AppId**, **AppName**, **TechnicalContact**, **Environment**, **DeployedStatus**, **MendixVersionOnMainline**, **DeployedMendixVersion**, **LastCommit**, **Members**, **Production**, **AppCreatedDate**, **CloudLocation**, **Fallback**, **CloudResourcePack**, **AppRAM**, **DbRAM**, **DbStorage**, **DbStorageExtra**, and **FileStorage**.
```
